src/main/java/jp/ikedam/jenkins/plugins/groovy_label_assignment/GroovyLabelAssignmentAction.java

```diff
@@ -32,16 +33,39 @@
  */
 public class GroovyLabelAssignmentAction implements LabelAssignmentAction
 {
-    private Label label;
+    @Deprecated
+    transient private Label label;
+    
+    private final String labelString;
     
     /**
      * Constructor
      * 
-     * @param label assigned label.
+     * @param labelString assigned label expression.
+     * @since 1.1.1
      */
+    public GroovyLabelAssignmentAction(String labelString)
+    {
+        this.labelString = labelString;
+    }
+    
+    /**
+     * @param label
+     * @deprecated use {@link GroovyLabelAssignmentAction#GroovyLabelAssignmentAction(String)}
+     */
+    @Deprecated
     public GroovyLabelAssignmentAction(Label label)
     {
-        this.label = label;
+        this(label.getExpression());
+    }
+    
+    private Object readResolve()
+    {
+        if(label != null)
+        {
+            return new GroovyLabelAssignmentAction(label.getExpression());

```

Comment:
I see the use of `label` here (and above) while `label` has been `@Deprecated`. Did you mean to replace these? Maybe call `getAssignedLabel`?